docs/src/modules/java-protobuf/pages/serving-web-resources.adoc

```diff
@@ -0,0 +1,91 @@
+= Serving Web Resources
+:page-supergroup-java-scala: Language
+
+This guide provides a simple example of how to serve arbitrary web resources like `.html` files.
```

Comment:
Wondering if we should have some kind of disclaimer about the fair / recommended usage of this feature. I think this is a very nice feature for fairly simple use cases, but we probably don't want to endorse it as a recomended way to build full-stack apps.. as in, Kalix is still pretty much focused on the API mainly.

Reply:
That makes sense. I will add an additional now. However, maybe we should re-think if this is even something that should be on the docs? 
"Here, look this feature in our docs but please do not use it" it kind of?

Reply:
I think a user can determine if this is a reasonable way for deploying their frontend or not themselves, and that it is fine as a doc section showing that it is one possible option.

Reply:
I definitely think it is worth in the docs (I was even thinking if this was worth it enough to add to the feature matrix but maybe not relevant enough..).

> I think a user can determine if this is a reasonable way for deploying their frontend or not themselves

Fair enough! 👍